src/auslib/web/admin/views/rules.py

```diff
@@ -260,6 +260,15 @@ def _post(self, transaction, changed_by):
             if what.get("fallbackMapping") is not None and len(fallback_mapping_values) != 1:
                 return problem(400, "Bad Request", "Invalid fallbackMapping value. No release name found in DB")
 
+            alias = what.get("alias", None)
+            if alias is not None and dbo.rules.getRule(alias):
+                return problem(400, "Bad Request", "Rule with alias exists.")
+
+            if alias is not None:
+                scheduled_rule_with_alias = dbo.rules.scheduled_changes.t.select().where(dbo.rules.scheduled_changes.base_alias == alias).execute().fetchall()
```

Comment:
This (and the similar block below) are missing the conditions I called out in https://github.com/mozilla-releng/balrog/pull/3010#discussion_r1362959833:
> * complete is False
> * change_type is insert or update